apps/common-app/src/examples/index.ts

```diff
@@ -130,7 +130,7 @@ import ComposedHandlerInternalMergingExample from './ComposedHandlerInternalMerg
 import BorderRadiiExample from './SharedElementTransitions/BorderRadii';
 import FreezingShareablesExample from './ShareableFreezingExample';
 import TabNavigatorExample from './SharedElementTransitions/TabNavigatorExample';
-import StrictDOMExample from './StrictDOMExample';
+// import StrictDOMExample from './StrictDOMExample';
```

Comment:
Do we want to keep it commented out? 😄

Reply:
Fixed in cf32307efcef94533b1a15581dc182614b51fe3a